include/lambdifier/binary_operator.hpp

```diff
@@ -26,11 +27,16 @@ class LAMBDIFIER_DLL_PUBLIC binary_operator
     // Getters/setters.
     const expression &get_lhs() const;
     const expression &get_rhs() const;
+    expression &access_lhs();
+    expression &access_rhs();
     void set_lhs(expression);
     void set_rhs(expression);
+    char get_op() const {return op;};
```

Comment:
Is this ``get_op()`` still necessary?

Reply:
I need it for the autodiff. The chain rules are different according to the operator.

Reply:
I will remove at a later stage if not neede.